seth/tts-CalculatorAssignment/JsKoo.js

```diff
@@ -0,0 +1,114 @@
+var onebtn = document.getElementById('calc-one');
+var onebtn = document.getElementById('calc-two');
+var onebtn = document.getElementById('calc-three');
+var onebtn = document.getElementById('calc-four');
+var onebtn = document.getElementById('calc-five');
+var onebtn = document.getElementById('calc-six');
+var onebtn = document.getElementById('calc-seven');
+var onebtn = document.getElementById('calc-eight');
+var onebtn = document.getElementById('calc-nine');
+var onebtn = document.getElementById('calc-zero');
+
+var decimalbtn = document.getElementById('decimal');
+var clearbtn = document.getElementById('clear');
+var backpacebtn = document.getElementById('backspace');
+var displayValElement = document.getElementById('Calculator-display');
+
+var displayVal = '0';
+var pendingVal;
+var evalStringArray = [];     /* problem here */
+
+var calcNumnbtns = document.getElementsByClassName('calc-btn-num');
+var calOperationBtns = document.getElementsByClassName('calc-btn-operator');
+
+var updateDisplayVal = (clickObj) => {
+  var btnText = clickObj.target.innerText;
+
+if(displayVal === '0')
+  displayVal = '';
+
+displayVal += btnText;
```

Comment:
Watch your spacing. Make sure spacing is consistent. Inside of of any function/block should be one tab. Also in your case statement below, they should all be spaced similarly vertically.